<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1208</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(38, 162, 105);
color: rgb(255, 255, 255);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="Title">
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Formulário de Consultoria</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>15</number>
          </property>
          <property name="topMargin">
           <number>15</number>
          </property>
          <property name="rightMargin">
           <number>15</number>
          </property>
          <property name="bottomMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Nome*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Name"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Email*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Email"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Telefone*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Telephone"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>Data da consulta*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDateEdit" name="Date">
                <property name="maximumDateTime">
                 <datetime>
                  <hour>23</hour>
                  <minute>59</minute>
                  <second>59</second>
                  <year>2025</year>
                  <month>12</month>
                  <day>31</day>
                 </datetime>
                </property>
                <property name="minimumDateTime">
                 <datetime>
                  <hour>0</hour>
                  <minute>0</minute>
                  <second>0</second>
                  <year>2022</year>
                  <month>1</month>
                  <day>2</day>
                 </datetime>
                </property>
                <property name="calendarPopup">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Status da consulta*</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="Status">
                <property name="currentText">
                 <string>Concluído</string>
                </property>
                <item>
                 <property name="text">
                  <string>Concluído</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Cancelado</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pendente</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Descrição*</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="Description"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="topMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QPushButton" name="BtnAdd">
              <property name="text">
               <string>Adicionar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BtnEdit">
              <property name="text">
               <string>Editar</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="BtnDelete">
              <property name="text">
               <string>Deletar</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="Table">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>130</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Email</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Telefone</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Estado</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descrição</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
